cloud/blockstore/libs/notify/https.cpp

```diff
@@ -939,10 +939,14 @@ void THttpsClient::Post(
     const TString& endpoint,
     const TString& data,
     const TString& contentType,
+    const std::optional<TString> iamToken,
```

Comment:
If the token is correct, no, but this can happen if there are any problems with the tokenClient. So, it seemed logical to separate case when the token is empty because of notify v1 and the case when the token client sends garbage.